getting-started/kubernetes/flannel/migration-from-flannel.md

```diff
@@ -1,64 +1,100 @@
 ---
-title: Migrate a cluster from flannel to Calico
-description: Upgrade from legacy flannel VXLAN to Calico VXLAN. 
+title: Migrate a Kubernetes cluster from flannel/Canal to Calico
+description: Preserve your existing VXLAN networking in Calico, but take full advantage of advanced network policy and other networking options.
 ---
 
-# About
+### Big picture
 
-This article describes how migrate an existing Kubernetes cluster with flannel networking to use Calico
-networking. Calico provides a migration tool that performs a rolling update of the nodes in the cluster.
-At the end, you will have a fully-functional Calico cluster using VXLAN networking between pods.
+Migrate an existing Kubernetes cluster with flannel/Canal to {{site.prodname}} networking. 
 
-# Prerequisites
+### Value
+
+If you are already using flannel for networking, it is easy to migrate to {{site.prodname}}'s native VXLAN networking. After migration, you can consider other {{site.prodname}} networking options like IP-in-IP or BGP peering. Either way, you automatically get {{site.prodname}} advanced network policy features. 
```

Comment:
> After migration, you can consider other {{site.prodname}} networking options like IP-in-IP or BGP peering.
I am not sure if we officially support switching from `vxlan` to `ip-ip` or `bgp peering`.  I feel it is better to replace `more network option` with `Calico IPAM` benefits (flexibilities, less ip space constraints etc.)

Reply:
Ah, I did not realize this limitation.

Reply:
Does this work now?

Reply:
Yes.